geocoder_tester/world/france/iledefrance/test_addresses.csv

```diff
@@ -512,9 +512,6 @@ query,lon,lat,limit,expected_name,expected_housenumber,expected_street,expected_
 12BIS Chemin d'Orveau Vayres-sur-Essonne,,,,12BIS Chemin d'Orveau,,,,91820
 12BIS Chemin d'Orveau 91820,,,,12BIS Chemin d'Orveau,,,Vayres-sur-Essonne,
 12BIS Chemin d'Orveau,2.351197,48.438203,,12BIS Chemin d'Orveau,,,,91820
-3 Avenue Lyautey Vert-le-Petit,,,,3 Avenue Lyautey,,,,91710

```

Comment:
According to the BAN, only the "3 avenue du Maréchal Lyautey 91179 Vert-le-Petit" exists: 

http://adresse.data.gouv.fr/map/#17/48.54230/2.36099

The BAN can be wrong, but I'd suggest following it unless proven its wrong. What do you think?